examples/mysql/main.tf

```diff
@@ -15,4 +15,5 @@ resource "humanitec_resource_definition_criteria" "mysql_basic" {
   resource_definition_id = module.mysql_basic.id
   app_id                 = humanitec_application.example.id
   class                  = "default"
+  force_delete           = true
```

Comment:
This is required as we otherwise have a circle here, the criteria waits for the app to be deleted (inside Humanitec) and TF can't delete the app if the criteria isn't deleted.